smriprep/utils/misc.py

```diff
@@ -21,3 +21,35 @@ def apply_lut(in_dseg, lut, newpath=None):
                    segm.affine, hdr).to_filename(out_file)
 
     return out_file
+
+
+def check_valid_fs_license():
```

Comment:
Are you planning on moving this function definition (`check_valid_fs_license`) to niworkflows? I think that was what I had in mind in first place. s/fMRIPrep will only need a quick call to it.